providers/fab/tests/unit/fab/auth_manager/models/test_user_model.py

```diff
@@ -0,0 +1,51 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import pytest
+
+pytestmark = pytest.mark.db_test
+
+try:
+    from airflow.providers.fab.auth_manager.models import User
+
+    class TestUserModelGetId:
+        def test_get_id_returns_str_for_int_id(self, session):
+            """
+            To ensure get_id() always returns a string.
+            This is required because return type str is expected.
+            """
+            user = User()
+            user.id = 999
+            result = user.get_id()
+
+            assert isinstance(result, str), f"Expected str, got {type(result)}"
+            assert result == "999"
+
+        def test_get_id_returns_str_for_str_id(self, session):
+            """
+            If the user id is a string, get_id() should return it without any modification.
+            """
+            user = User()
+            user.id = "999"
+            result = user.get_id()
+
+            assert isinstance(result, str)
+            assert result == "999"
```

Comment:
How about refactoring the test cases with `parametrize` ?

```suggestion
    @pytest.mark.parametrize(
        "user_id,expected",
        [
            (999, "999"),
            ("999", "999"),
        ]
    )
    def test_get_id_returns_str(self, session, user_id, expected):
        """
        Ensure get_id() always returns a string representation of the id.
        """
        user = User()
        user.id = user_id
        result = user.get_id()

        assert isinstance(result, str), f"Expected str, got {type(result)}"
        assert result == expected
```

Reply:
I also don't think we need to try catch the whole block. we can simply include the import part and skip when the exception is caught

Reply:
+1